packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -54,6 +57,30 @@ const progress = (url) => {
   })
 }
 
+const followProgress = (jobId, client, output) => {
+  const spinner = output
+    .spinner({
+      text: `~0% done.`,
```

Comment:
Nitpick: This looks like a false promise to me, using the word `done`. It's common english expression of mentioning progress but an explicit word like `progress: 0%` may be the other option. (Job may fail for various reaons.)

Reply:
Good point!

Reply:
Changed it a bit in 8c8b25e88. Hope that's better 🙂

![recording_2020-11-27-10:08:44](https://user-images.githubusercontent.com/3595094/100431687-cd275000-3098-11eb-9534-af6be93cde76.gif)